### DISCUSSION TITLE
Documentation on clusters

### DISCUSSION THREAD

**Discussion by nvanggaard** [Koenkk/zigbee2mqtt#10726] [General]
Hi

I have searched everywhere (even page 3 on google), and I'm not able to find any information about cluster when binding two devices.

I can figure out myself what onOff and Level does. And i think Scenes are to switch between scenes (Is this correct? - I cannot get it to work') and what about the rest, is there any documentation about these clusters anywhere?

Best regards 
Nichlas

**Comment by sjorge** [comment]
They are documented in the ZCL: https://github.com/Koenkk/zigbee-herdsman/blob/master/docs/07-5123-08-Zigbee-Cluster-Library.pdf

Note that sadly not all device strictly follow the ZCL, so there may be some odd broken things out there.

**Reply by nvanggaard** [reply]
Well, from the documentation i still don't get how I get the scene cluster to work in z2m. 
Any one who can help ?

**Comment by tomek27** [comment]
Bumping up this one. I am also looking for informations how clusters are converted into topics.
For context I am building custom solution based on ESP32 where I want to send values from 0 - 100
Tried with multistate and analog clusters from ZCL but when device appears even if it see these clusters I see nothing in exposes tab.
I could use level but this is rather for bulb brightens.
